cookbooks/fb_users/resources/default.rb

```diff
@@ -136,9 +149,12 @@
       manage_home manage_homedir
       home homedir
       comment mapinfo['comment'] if mapinfo['comment']
-      password pass if pass
-      if FB::Version.new(Chef::VERSION) >= FB::Version.new('15')
-        secure_token info['secure_token'] unless info['secure_token'].nil?
+      if pass && set_passwords
+        password pass
+      end
+      if FB::Version.new(Chef::VERSION) >= FB::Version.new('15') &&
+          !mapinfo['secure_token'].nil?
+        secure_token info['secure_token']
```

Comment:
mapinfo['secure_token'], right?

Reply:
Hmmm ... we checked if `info` was nil before, now checking `mapinfo`; not sure what's intended here.

Reply:
yeah this is weird, lemme fix

Reply:
Fixed. Nice catch.